### PR TITLE
Prevent the example of 'nan' result from eval() from breaking tests

### PR DESCRIPTION
**Pull request by vpodzime**
-1^2.1 can be -nan on some platforms.

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a PR! Maybe @olehermanse can review this?

**Comment by vpodzime** [comment]
Unfortunately, all variants differ between nested VMs and non-nested VMs.

**Comment by vpodzime** [comment]
@cf-bottom jenkins with exotics, please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/2480/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/2480/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-2480/

File: examples/eval.cf [hunk 1]
```diff
@@ -19,7 +19,7 @@ bundle agent run
       "values[6]" string => ""; # 0
       "values[7]" string => "3 / 0"; # inf but not an error
       "values[8]" string => "3^3";
-      "values[9]" string => "-1^2.1"; # -nan but not an error
+      # "values[9]" string => "-1^2.1"; # 'nan' or '-nan' (on some platforms)
```
Comment (nickanderson) [inline examples/eval.cf]: +1

**Pull request merged**